README.md

```diff
@@ -464,6 +464,12 @@ For complex products, like technology platforms, it's a best practice for the de
 - [Communities aren't funnels](https://www.youtube.com/watch?v=EDgNuFdVay0) — Josh Dzielak, DevRelCon 2019
 - [How we're using Orbit to build Orbit](https://orbit.love/blog/how-we-use-orbit-to-build-orbit/) - Orbit Blog
 
+<p align="left">
+  <img alt="Orbit-themed Slack icons" src="images/orbit-model-slack-icons.png" width="360">
+</p>
+
+[Download our custom Orbit Model icons for Slack!](https://orbit.love/blog/download-these-orbit-model-icons-for-your-slack-workspace)
```

Comment:
@alexlsalt Since folks can use these for Discord or other tools, maybe we just call them "Orbit emojis" or "Orbit chat icons"?

Reply:
100%